hoomd/mpcd/pytest/test_snapshot.py

```diff
@@ -0,0 +1,261 @@
+# Copyright (c) 2009-2023 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+import hoomd
+from hoomd import Simulation
+import numpy as np
+import pytest
+
+# default testing configuration, used in setting values below
+test_positions = np.array([[1, 2, 3], [-9, -6, -3], [7, 8, 9]])
+test_velocities = np.array([[-1, -2, -3], [4, 5, 6], [-7, -8, -9]])
+test_typeids = np.array([1, 0, 2])
+test_types = ["M", "P", "H"]
+test_mass = 1.5
+
+
+@pytest.fixture
+def snap():
+    snap_ = hoomd.Snapshot()
+    snap_.configuration.box = [20, 20, 20, 0, 0, 0]
+    return snap_
+
+
+def test_make(snap):
```

Comment:
```suggestion
def test_get_set_mpcd_data(snap):
```
I would like to see this test have a name more specific to what its doing.